common/assets/scss/components/_icons.scss

```diff
@@ -0,0 +1,25 @@
+$width: 20px;
+
+.app-icon {
+  display: inline-block;
+  vertical-align: middle;
+  background-repeat: no-repeat;
+  background-size: $width $width;
+  padding-left: $width + govuk-spacing(2);
```

Comment:
Looks like we just need to add `height: $width`. In fact, maybe something like this:

```scss
$_icon-width: 20px;
$_icon-height: 20px;

...
background-size: $_icon-width $_icon-height;
padding-left: $_icon-width + govuk-spacing(2);
height: $_icon-height;
...
```